src/table/interfaces.tsx

```diff
@@ -228,6 +228,22 @@ export interface TableProps<T = any> extends BaseComponentProps {
    * @visualrefresh `embedded`, `stacked`, and `full-page` variants
    */
   variant?: TableProps.Variant;
+
+  /**
+   * Use this property to inform screen readers how many items there are in a table.
+   * It specifies the total count of all items in a table.
+   * If there is an unknown total of items in a table, leave this property undefined.   */
+  totalItemsCount?: number;
+  /**
+   *  Use this property to inform screen readers which range of items is currently displayed in the table.
+   *  It specifies the index (1-based) of the first item in the table.
+   *  If the table has no pagination, leave this property undefined.   */
+  firstIndex?: number;
+  /**
+   * Use this function to announce page changes for screen reader users.
+   * When firstIndex is available, it returns a text when the firstIndex or totalItemsCount change.
```

Comment:
The wording is a bit confusing to me. You use "when" twice.

> it returns a text when the firstIndex or totalItemsCount change

I would think that the function *always* returns a string, not just when the properties change.

Reply:
reading it again I agree, what about: 

> If firstIndex is available, it announces a text when the firstIndex or totalItemsCount change.

Reply:
A lot better! I'd be fine with it, but I feel like we are still explaining the internals in too much detail. Our customers should already trust us to announce the text at the right time, but I do think mentioning the required properties is useful.

How about:

> Use this function to announce page changes to screen reader users.
> Requires the properties `firstIndex` and `totalItemsCount` to be set correctly.